src/Microsoft.IIS.Administration.WebServer/Microsoft.IIS.Administration.WebServer.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\sign.props" />
   <Import Project="..\..\build\version.props" />
+  <Import Project="..\..\build\plugins.props" />
 
   <PropertyGroup>
     <Description>Microsoft.IIS.Administration.WebServer Class Library</Description>
     <VersionPrefix>$(IISAdministrationVersion)</VersionPrefix>
     <Authors>Microsoft</Authors>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
as you have so many satellite projects, you can aggregate the common properties into one props file. This will save  future maintenance effort.

Reply:
So, 2 things I tried
1. put `TargetFramework` property in `plugins.props`
2. Declare a variable `IISAdministrationTargetFramework` in `plugins.props`.

When running `dotnet restore`, both scenarios run into the same error:
`C:\Program Files\dotnet\sdk\2.1.603\NuGet.targets(119,5): error : Invalid framework identifier ''.`

Looks like dotnet restore does not know how to resolve project imports. Refactoring seems not possible

Reply:
Created https://github.com/Microsoft/IIS.Administration/issues/240